### PR TITLE
Fix upload styling

### PR DESCRIPTION
**Pull request by inikolova**
Fix broken appearance of the upload component

### CODE REVIEW

File: scss/upload/_layout.scss [hunk 1]
```diff
@@ -139,60 +147,58 @@
...
+
         .k-upload-status {
-            display: flex;
-            align-items: center;
-            position: relative;
-            margin-left: auto;
+            position: absolute;
+            line-height: 1em;
+            right: $padding-x;
+            top: $padding-x + $padding-y;
```
Comment (gyoshev) [inline scss/upload/_layout.scss]: Why does horizontal padding influence the `top` position? Shouldn't that be `(multiplier) * $padding-y`, by any chance?

File: scss/upload/_layout.scss [hunk 2]
```diff
@@ -210,8 +216,9 @@
     }
 
     .k-file-name-size-wrapper {
-        vertical-align: middle;
-        margin-left: 1em;
+        display: block;
+        margin-left: calc(#{$padding-x} + #{40px});
+        margin-right: calc(#{$padding-x} + #{16px + #{$padding-x-sm}});
```
Comment (gyoshev) [inline scss/upload/_layout.scss]: What are `40px` and `16px`? How are these related to the surrounding code?

File: scss/upload/_theme.scss [hunk 3]
```diff
@@ -11,18 +11,35 @@
     $upload-success-bg: $success !default;
     $upload-disabled-text: darken($widget-bg, 27%) !default;
     $upload-disabled-border: darken($widget-bg, 27%) !default;
+    $upload-focused-status-box-shadow: 0 0 0 2px darken($input-bg, 6%) !default;
```
Comment (gyoshev) [inline scss/upload/_theme.scss:14]: How about naming this `$upload-focused-box-shadow`?

File: scss/upload/_layout.scss [hunk 4]
```diff
@@ -1,43 +1,43 @@
...
     }
 
     .k-upload {
-        border-width: 1px;
-        border-style: solid;
-
-        .k-action-buttons {
-            margin: -6px 0 0;
+        .k-upload-button {
+            min-width: 6.643em;
```
Comment (gyoshev) [inline scss/upload/_layout.scss]: Oddly specific. Why exactly this value?

File: scss/upload/_layout.scss (hunk 2 above)
Comment (inikolova) [reply scss/upload/_layout.scss]: There are as follows:
-40px - width of the uploaded file image;
-16px - clear icon width

File: scss/upload/_theme.scss (hunk 3 above)
Comment (inikolova) [reply scss/upload/_theme.scss:14]: This shadow is applied to the focused status icon only. Changing the var name to $upload-focused-box-shadow assumes that it will be applied to the Upload widhet itself. @gyoshev what do you think?

File: scss/upload/_layout.scss (hunk 1 above)
Comment (inikolova) [reply scss/upload/_layout.scss]: I agree. Done

File: scss/upload/_layout.scss (hunk 2 above)
Comment (gyoshev) [reply scss/upload/_layout.scss]: The icon size is available through the `$icon-size` variable, so using that will explain the intent.
The uploaded file image size seems computed from the `$padding-y-lg` and the width of `.k-file-extension-wrapper`. Using variables will continue to work after their values are changed, for example in a custom theme.

File: scss/upload/_layout.scss (hunk 4 above)
Comment (inikolova) [reply scss/upload/_layout.scss]: After further discussion with @yordanov I changed the upload min-width

File: scss/upload/_layout.scss (hunk 2 above)
Comment (inikolova) [reply scss/upload/_layout.scss]: You're right - I will make the change
